pilot/pkg/xds/ads.go

```diff
@@ -818,9 +818,11 @@ func AdsPushAll(s *DiscoveryServer) {
 func (s *DiscoveryServer) AdsPushAll(version string, req *model.PushRequest) {
 	// If we don't know what updated, cannot safely cache. Clear the whole cache
 	if len(req.ConfigsUpdated) == 0 {
+		adsLog.Infof("drop cache all")
```

Comment:
Do you want to keep these info logs?

Reply:
definitely not, sorry this PR was meant to be marked as WIP

Reply:
np